www/src/components/landingPage/ClipboardSelect.tsx

```diff
@@ -0,0 +1,118 @@
+import { Menu, Transition } from "@headlessui/react";
+import clsx from "clsx";
+import { Fragment, useState } from "react";
+
+const commands = [
+  {
+    command: "npm create t3-app@latest",
+    manager: "npm",
+  },
+  {
+    command: "yarn create t3-app",
+    manager: "yarn",
+  },
+  {
+    command: "pnpm create t3-app@latest",
+    manager: "pnpm",
+  },
+  {
+    command: "bunx create-t3-app@latest",
```

Comment:
bun?

Reply:
in the README.md it says `bunx`

Reply:
https://github.com/t3-oss/create-t3-app#bun

Reply:
That should be updated too

Reply:
okay I can add that to this PR then? Updating now!

Reply:
Added!